replay-scripts/src/main/python/replay/data_replay.py

```diff
@@ -0,0 +1,97 @@
+import os
+import sys
+import findspark
+import argparse
+from pyspark.sql import SparkSession
+from pyspark.sql import functions as func
+from datetime import date, timedelta, datetime
+from pathlib import Path
+from resources import replay_config
+from util.azure_utils import copy_data, delete_data
+from util.postgres_utils import executeQuery
+from util.replay_utils import push_data, getDates, getBackUpDetails, getKafkaTopic, getInputPrefix, restoreBackupData, backupData, deleteBackupData, getFilterStr, getFilterDetails
+
+start_time = datetime.now()
+print("Started at: ", start_time.strftime('%Y-%m-%d %H:%M:%S'))
+parser = argparse.ArgumentParser()
+parser.add_argument("container", type=str, help="the data container")
+parser.add_argument("prefix", type=str, help="replay data prefix")
+parser.add_argument("start_date", type=str, help="YYYY-MM-DD, replay start date")
+parser.add_argument("end_date", type=str, help="YYYY-MM-DD, replay end date")
+parser.add_argument("kafka_broker_list", type=str, help="kafka broker details")
+parser.add_argument("delete_backups", type=str, default="False", help="boolean flag whether to delete backups")
+
+args = parser.parse_args()
+container = args.container
+prefix = args.prefix
+start_date = args.start_date
+end_date = args.end_date
+kafka_broker_list = args.kafka_broker_list
+delete_backups = args.delete_backups
+
+config_json = replay_config.init()
+
+dateRange = getDates(start_date, end_date)
+try:       
+    filterString = getFilterStr(getFilterDetails(config_json, prefix))
+    for date in dateRange:
+        try:
```

Comment:
Probably introduce a wait/sleep time for each day replay starting with 30 mins. Ideally would like to check the offset and lag and create a rule on top of them